index.js

```diff
@@ -44,6 +44,14 @@ var isArray = Array.isArray;
 // interdependence between different functions.
 var conjure;
 
+// a special hasOwnProperty function so we can override the built-in one
+
+var hasOwnProperty = Object.prototype.hasOwnProperty;
+
+function hop(obj, propName) {
```

Comment:
Please find another name, so that we don't have to read that definition to be able to read the code.
Something more verbose.
ie:
* `_ hasOwnProperty`
* `Tome.hasOwnProperty`
* ...